Rakefile

```diff
@@ -6,6 +6,21 @@ def usage
     Usage:
       bundle exec rake
 
+      There are three environment variables that control how the tests are run.
+
+      TEST_REPO: This is the repository which will be tested.
+      CORE_REPO: If TEST_REPO is a plugin and you need to specify a different core branch
+      GEM_REPOS: Optional a comma-separated-list of other plugins/gems which are needed to run the tests
+
+      All of these support the following formats:
+      Remote: [Org/]Repostory[@Ref|#PR]
+       Org:        Optional, defaults to ManageIQ
+       Repository: Required, the name of the repository
+       @Ref:       Optional, defaults to master. Can be any ref (branch, tag, or SHA). Mutually exclusive with #PR
+       #PR:        Optional, references a pull-request number.  Mutually exclusive with @Ref
+
+      Local: Either a fully qualified path or a relative path e.g. /path/to/repo, ~/relative/to/home, ../relative/to/current/dir
+
     Examples:
     # Test a specific plugin against ManageIQ master
     TEST_REPO=manageiq-ui-classic bundle exec rake
```

Comment:
Minor, but can we indent these example 2 spaces under the Examples: header?  (maybe separate PR)

Reply:
Yes, done